### DISCUSSION TITLE
Integration not appearing

### DISCUSSION THREAD

**Discussion by bazyrush** [ofalvai/home-assistant-candy#48] [General]
Hi, I'm struggling to install this -- I've added the custom component from HACS, restarted HA, but can't find the integration under config / integrations. Am I missing something?

**Comment by ofalvai** [comment]
I'm not sure what is the problem, do you see any reference to "candy" in the logs maybe?

**Comment by Jeronimo2022** [comment]
I'm having the same issue, I Have the latest Home Assistant version + the latest HAC S - I did the instalation today, and Candy doesnt show up??? any idea?

**Comment by ofalvai** [comment]
Are you able to install other custom integrations through HACS?

**Reply by Jeronimo2022** [reply]
I tried to install others and I can't. I  have the HASS version - 2021.12.8 - so I thin HAS doesn't work on it. But the integration metioned candy simplify even doesn't show up on the list.

I think I need to install a previous version of HASS. Yes?

**Reply by Jeronimo2022** [reply]
Problem solve - It was related with my first intalation issue. So HACS was delayed

**Reply by similair** [reply]
I have the same problem, anyone have a definitive answer?
Many thanks
p.s.
Just to be clear I have latest HACS and HA. Installed as per instructions, restarted, go to settings/integrations click the big blue Add Integration and search Candy, nothing appears.

Thanks

**Comment by similair** [comment]
I have the same problem, anyone have a definitive answer?
Many thanks
p.s.
Just to be clear I have latest HACS and HA. Installed as per instructions, restarted, go to settings/integrations click the big blue Add Integration and search Candy, nothing appears.

Thanks
